src/functions/env.ts

```diff
@@ -240,6 +240,24 @@ export function hasUserEnvs({
   return findEnvfiles(functionsSource, projectId, projectAlias, isEmulator).length > 0;
 }
 
+/* eslint-disable */
```

Comment:
Why are you disabling this?

Reply:
eslint is extremely not happy about the function parameters not being used anywhere in the body of the function, I think.